sam/cigar.go

```diff
@@ -52,18 +52,19 @@ func (c Cigar) String() string {
 	return b.String()
 }
 
-// RefLen returns the total number of Reference bases consumbed by the Cigar
-func (c Cigar) RefLen() int {
-	var pos, end int
+// Lengths returns the total number of Reference, and Read bases consumbed by the Cigar
```

Comment:
`// Lengths returns the number of reference and read bases described by the Cigar.`